common-components/index.md

```diff
@@ -461,6 +461,27 @@ SDT have also set up a Dynamics Community of Practice, bringing together teams i
 
 For more information, or if you'd like to talk through about options for CRM, please contact [Iain McIntosh](https://gbr.delve.office.com/?u=b7680d65-eb71-4c1c-a68e-b95fa6248026&v=work).
 
+
+<a name="ccts"></a>
+### Contact Centre Technology Service (CCTS)
+CCTS provides a shared contact centre telephony capability with associated services such as non-geographic numbers, Interactive Voice Response (IVR), call management, recording and reporting. It is currently used by the following DfE service lines. 
+- Ministerial and Parliamentary Communication Division (MPCD) 
+- Teachers Qualification Unit (TQU) 
+- DfE Service Desk 
+- ESFA Customer Services 
+- Human Resources 
+- Press Office 
+- Standards and Testing Agency (STA) 
+
+Initial (Phase 1) deployment of CCTS focused on providing core contact centre telephony capabilities. Future phases will focus on integration with other common components and delivery of additional communication channels as shown below. 
+
+![ccts-phases](../images/ccts-phases.png)
```

Comment:
As the image contains quite a bit of info, would be good to reference this in alt text, e.g. 

`![ccts-phases](../images/ccts-phases.png "Image showing capabilities and delivery phases for CCTS")`